ghrunner/config/crd/bases/ghrunner.ghrunner_ghrunners.yaml

```diff
@@ -41,23 +41,27 @@ spec:
             properties:
               owner:
                 description: Owner defines the owner of the repository
+                minLength: 1
                 type: string
               pat:
                 description: 'Pat defines the personal access token: Fine grained
                   permissions for read/write on administration'
+                minLength: 1
                 type: string
               repo:
                 description: Repo defines the repository name
+                minLength: 1
                 type: string
               size:
-                description: |-
```

Comment:
Any particular reason for omitting the description for size?
if this is still a field in our CRD , I think it should have a description?